.github/workflows/beaker.yml

```diff
@@ -162,6 +162,13 @@ jobs:
     name: "${{ matrix.name }}"
     steps:
       - uses: actions/checkout@v4
+      - name: Setup podman
+        if: ${{ inputs.beaker_hypervisor == 'container_podman' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y podman crun
```

Comment:
not even sure this is needed -- I think podman is installed by default on 24.04 boxen

Reply:
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md says it's installed,